Automattic/jetpack
# Admin components: Add default links to logo footers (Jetpack's and Automattic's)

Fixes #23108
Fixes #23112

#### Changes proposed in this Pull Request:

* Add new prop to add a link to the Module name in the JetpackFooter defaulting to `https://jetpack.com`
* Add link to jetpack.com in My Jetpack footer
* Makes the A8C logo default to linking to `https://automattic.com`
* Updates components to deal with the new defaults.

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Open Storybook
* Go to Js packages > Components > Jetpack Footer
* Check there's a new `moduleNameHref` prop in the Contorls tab
* Make sure no `a` tag is rendered in the module name when this prop is empty
* Make sure a `a` tag is properly rendered when there is a value
* Make sure styles look good
<img width="784" alt="Screen Shot 2022-03-03 at 9 29 12 AM" src="https://user-images.githubusercontent.com/77539/156565053-87652386-a3a9-4c2d-b60a-222634a4d160.png">


Now on My Jetpack
* Visit My jetpack
* make sure the "Jetpack" in the footer links to jetpack.com
* open the interstitial pages and check the same thing

Note 1: I didn't make this link the default because jetpack.com is the default link to the a8c logo already (shouldn't it be a8c.com??)
